src/pages/ProductDetail/components/ProductInfo/ProductCategory/ProductCategory.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './ProductCategory.scss';
+
+class ProductCategory extends Component {
+  constructor() {
+    super();
+    this.state = {
+      focus: '',
+    };
+  }
+
+  setFocusCategory = category => {
+    const { focus } = this.state;
+    focus === category
+      ? this.setState({
+          focus: '',
+        })
+      : this.setState({
+          focus: category,
+        });
+  };
+
+  render() {
+    const { focus } = this.state;
+    const { name, contextList } = this.props;
+
+    return (
+      <div className="productCategory">
+        <div
+          className={`${focus === name && 'focus'} category`}
+          name={name}
+          onClick={() => this.setFocusCategory(name)}
+        >
+          <div className="header">
+            <span>{name}</span>
+            <span>{focus === name ? '-' : '+'}</span>
+          </div>
+          <div className="context">
+            {Array.isArray(contextList) ? (
+              contextList.map((context, idx) => (
+                <div
+                  className={name === 'infomation' ? 'contant' : undefined}
```

Comment:
```suggestion
                  className={name === 'infomation' ? 'contant' : ''}
```
- undefined는 개발자가 직접 넣어주는 값이 아니라 예상치 못한 경우에 대해 시스템이 리턴하는 값잆니다. 상황만 보면 null이 더 맞는거 같습니다
- 다만 이렇게 되면 className="null" 처럼 될 것이기 때문에 그냥 빈 문자열을 넣어주시는게 낫겠습니다

Reply:
수정했습니다